integration-test/src/test/java/org/apache/iotdb/db/it/udf/IoTDBUDFManagementIT.java

```diff
@@ -16,77 +16,69 @@
+@RunWith(IoTDBTestRunner.class)
+@Category({LocalStandaloneIT.class, ClusterIT.class})
 public class IoTDBUDFManagementIT {
 
   private static final int NATIVE_FUNCTIONS_COUNT =
-      BuiltinAggregationFunction.getNativeFunctionNames().size();
+      BuiltinAggregationFunctionEnum.getNativeFunctionNames().size();
   private static final int BUILTIN_FUNCTIONS_COUNT =
-      BuiltinTimeSeriesGeneratingFunction.values().length;
+      BuiltinTimeSeriesGeneratingFunctionEnum.values().length;
+
+  private static final String FUNCTION_TYPE_NATIVE = "native";
+  private static final String FUNCTION_TYPE_BUILTIN_UDTF = "built-in UDTF";
+  private static final String FUNCTION_TYPE_EXTERNAL_UDTF = "external UDTF";
+
+  protected static boolean enableSeqSpaceCompaction;
+  protected static boolean enableUnseqSpaceCompaction;
+  protected static boolean enableCrossSpaceCompaction;
 
   @Before
   public void setUp() throws Exception {
-    EnvironmentUtils.envSetUp();
-    IoTDB.schemaProcessor.setStorageGroup(new PartialPath("root.vehicle"));
-    IoTDB.schemaProcessor.createTimeseries(
-        new PartialPath("root.vehicle.d1.s1"),
-        TSDataType.FLOAT,
-        TSEncoding.PLAIN,
-        CompressionType.UNCOMPRESSED,
-        null);
-    IoTDB.schemaProcessor.createTimeseries(
-        new PartialPath("root.vehicle.d1.s2"),
-        TSDataType.FLOAT,
-        TSEncoding.PLAIN,
-        CompressionType.UNCOMPRESSED,
-        null);
-    Class.forName(Config.JDBC_DRIVER_NAME);
+    enableSeqSpaceCompaction = ConfigFactory.getConfig().isEnableSeqSpaceCompaction();
+    enableUnseqSpaceCompaction = ConfigFactory.getConfig().isEnableUnseqSpaceCompaction();
+    enableCrossSpaceCompaction = ConfigFactory.getConfig().isEnableCrossSpaceCompaction();
+    ConfigFactory.getConfig().setEnableSeqSpaceCompaction(false);
+    ConfigFactory.getConfig().setEnableUnseqSpaceCompaction(false);
+    ConfigFactory.getConfig().setEnableCrossSpaceCompaction(false);
+    EnvFactory.getEnv().initBeforeClass();
   }
 
   @After
-  public void tearDown() throws Exception {
-    EnvironmentUtils.cleanEnv();
+  public void tearDown() {
+    EnvFactory.getEnv().cleanAfterClass();
+    ConfigFactory.getConfig().setEnableSeqSpaceCompaction(enableSeqSpaceCompaction);
+    ConfigFactory.getConfig().setEnableUnseqSpaceCompaction(enableUnseqSpaceCompaction);
+    ConfigFactory.getConfig().setEnableCrossSpaceCompaction(enableCrossSpaceCompaction);
   }
 
   @Test
   public void testCreateReflectShowDrop() {
-    try (Connection connection =
-            DriverManager.getConnection(
-                Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root", "root");
+    try (Connection connection = EnvFactory.getEnv().getConnection();
         Statement statement = connection.createStatement()) {
       statement.execute("create function udf as 'org.apache.iotdb.db.query.udf.example.Adder'");
       statement.execute("select udf(*, *) from root.vehicle");
```

Comment:
Why executing query sql here, not verifying the correctness?

Reply:
I think query here is to make sure that udf function itself is created correctly, So in this IT we don't need to check the result of this sql.